src/Codeception/Subscriber/Console.php

```diff
@@ -353,12 +376,8 @@ protected function printException($e, $cause = null)
 
         if ($e instanceof \PHPUnit_Framework_ExpectationFailedException) {
             if ($e->getComparisonFailure()) {
-                $comp = $e->getComparisonFailure();
-                if ($comp->getDiff()) {
-                    $message = $this->message($comp->getMessage())->append(" ( <comment>-Expected</comment> | <info>+Actual</info> ) \n");
-                    $message->append("- <comment>" . str_replace("\n", "\n- ", $comp->getExpectedAsString()))->append("</comment>\n");
-                    $message->append("+ <info>" . str_replace("\n", "\n+ ", $comp->getActualAsString()))->append("</info>\n");
-                }
+                $comparisionFailure = $e->getComparisonFailure();
```

Comment:
Microoptimization 2. Could you make this assignment above if and use variable in if?